components/src/core/Drawer/Drawer.tsx

```diff
@@ -20,7 +20,7 @@ const useStyles = makeStyles({
 });
 
 type DrawerClasses = {
-    root?: string;
```

Comment:
We shouldn't get rid of the root class...we should use it on the MUI drawer....we should be able to do `root` the same was as we are doing `paper`